DayrSuryan/DSEthiop13.xml

```diff
@@ -76,15 +80,13 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </hi> ዝቅ፡ ዘወጽአ፡ እምጐንድር፡ እምዮሐንስ፡ እስከ፡ ሁሐንስ<hi rend="rubric">፨</hi> ለገቢዋቲክሙ፡ መዋ፡  <hi rend="rubric">
                            <gap reason="lost"/>
                               ኒ፡ ሐፃን፡ ነቢየ፡ ልዑ<gap reason="lost"/>
-                        </hi>
-                        መ፡ ተሐውር፡ ቅድመ፡ እግዚአብሔር፡ ከመ፡ ተጸሕ፡ ፍኖቶ<hi rend="rubric">፨</hi>
+                        </hi> መ፡ ተሐውር፡ ቅድመ፡ እግዚአብሔር፡ ከመ፡ ተጸሕ፡ ፍኖቶ<hi rend="rubric">፨</hi>
                         <hi rend="rubric">ሰላ፡ ለአብ፡ ገባ<supplied reason="lost" resp="DR">ሬ</supplied>፡</hi>
-                        ኵሉ፡  ዛቲ፡ ያ፡ እቲ፡ ትንቢቶሙ፡ ለነቢያት፡ ስብከቶሙ፡ ለሐዋርያት፡ እሞሙ፡ ለሰማእት፡ ወእኅቶሙ፡ ለመላእክት። ዓዲ፡ ግነዩ፡ አርያ፡ ሃሌ፡ ሉያ፡ ለአብ፡ ቃለ፡ ዓዋዲ፡ ዘይሰብክ፡ በገዳም፡ <hi rend="rubric">ሃሌ፡
+                        ኵሉ፡  ዛቲ፡ ይእቲ፡ ትንቢቶሙ፡ ለነቢያት፡ ስብከቶሙ፡ ለሐዋርያት፡ እሞሙ፡ ለሰማእት፡ ወእኅቶሙ፡ ለመላእክት። ዓዲ፡ ግነዩ፡ አርያ፡ ሃሌ፡ ሉያ፡ ለአብ፡ ቃለ፡ ዓዋዲ፡ ዘይሰብክ፡ በገዳም፡ <hi rend="rubric">ሃሌ፡
```

Comment:
are you sure ያ፡ እቲ፡ is not written in the manuscript? We don't have the image of this page, I think?

Reply:
We have, it is Ya.